index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portifolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:ital,wght@1,500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="first-page">
+      <header>
+        <div class="logo"><a href="">My Logo</a></div>
+        <nav class="menu">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </nav>
+      </header>
```

Comment:
- [ ] Kindly increase the spacing around the header section to match the given template.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/Etsubdink-Demeke/portifolio/assets/62290545/91682181-268f-458d-9e0d-948409871ec2) | ![image](https://github.com/Etsubdink-Demeke/portifolio/assets/62290545/dabd813a-ad82-4fb9-9c70-a3d8cd8c30f2) |